### DISCUSSION TITLE
Wiki JS - What am i doing wrong arrrrrrrr..

### DISCUSSION THREAD

**Discussion by TheWizardUk** [requarks/wiki#4100] [Help / Questions]
Hi

I have done some testing with ubuntu, centos  and a centos / docker mix with various versions of node.js and postgresql. All within the support versions detailed on the wiki js site and presented with the same issue each time.

My issue, due to corporate network restrictions only port 80 is open and already in use by other applications so I am using proxypass, example below I have removed some personal details. 

<VirtualHost *:80
  ProxyPreserveHost on
  ProxyPass /wiki http://localhost:3000 retry=1 acquire=3000 timeout=600 keepalive=On                 
  ProxyPassReverse /wiki http://localhost:3000
  RequestHeader set Host localhost
</VirtualHost>

When hitting the url I am faced with just a white page.  I can see a header subject “wiki setup” or the title of my homepage I created below, so I know it’s  kind of working, and If I inspect the page with some browser tools I can see the context 

Example, If I RDP to an environment  where theses network restrictions are not in place I can access the wiki js without any issues via port 3000

Creating a home page with some duff info and saving it  in the RDP env then going back to the corporate network, although still a white screen inspecting the source I can see the changes made in the RDP session.

I have tried enabling / disabling some of the wiki js security setting x fow etc still no luck 

Am I doing something wrong? 

Thanks for your time

**Comment by loyalstr** [comment]
hope these helps

[https://docs.requarks.io/en/troubleshooting](https://docs.requarks.io/en/troubleshooting)
